<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>idsEditorWindow</class>
 <widget class="QMainWindow" name="idsEditorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>860</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>97</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>970</width>
    <height>860</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>970</width>
    <height>860</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IDS Editor Window</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QMdiArea" name="mdi_list">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>19</y>
      <width>800</width>
      <height>835</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>835</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>800</width>
      <height>835</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>130</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="lbl_IDSEditor_title">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>73</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>128</width>
         <height>73</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>IDS Editor</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ids_info">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>128</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>General Information</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ids_specifications">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>128</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>Specifications</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ids_audit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>128</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>Audit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_ids_save">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>128</horstretch>
         <verstretch>28</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>128</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>Save IDS</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QMdiArea" name="mdi_editor">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>810</width>
      <height>610</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>810</width>
      <height>310</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>810</width>
      <height>610</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_back">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>128</width>
      <height>28</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>128</width>
      <height>28</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>128</width>
      <height>28</height>
     </size>
    </property>
    <property name="text">
     <string>Back to IDS Manager</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
